pkg/api/types.go

```diff
@@ -1662,6 +1665,27 @@ type SecretList struct {
 	Items []Secret `json:"items"`
 }
 
+// SecurityContextSpec holds security configuration that will be applied to the pod's containers
+type SecurityContextSpec struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
```

Comment:
Understood.  I wanted to get some confirmation on the types, which was good since they were all wrong :) Once those are done I'll make sure all the types, fuzzer, and conversion updates are correct.